src/jqLite.js

```diff
@@ -54,7 +54,7 @@
  *
  * - [`addClass()`](http://api.jquery.com/addClass/) - Does not support a function as first argument
  * - [`after()`](http://api.jquery.com/after/)
- * - [`append()`](http://api.jquery.com/append/)
+ * - [`append()`](http://api.jquery.com/append/) (contrary to jQuery, this doesn't clone elements so will not work correctly when invoked on a jqLite object containing more than one DOM node)
```

Comment:
Could you break the line at 100 characters and use the same format as the other notes, i.e. `- contrary ...` instead of `(contrary ..)

Reply:
We don't break jqLite method descriptions at all, see some of the descriptions below. I changed the format to match others, PR updated.

Reply:
It's still easier to read them when they break at 100 chars. And I think it doesn't affect the eventual layout. And it also makes @gkalpak sad 😁

Reply:
Updated but note that in the jqLite methods list 9 lines already don't fit in the 100 chars limit. :P

Reply:
I'll just add I'd never want to make @gkalpak sad!

Reply:
This has been a real emotional roller-coaster for me. But I'm very excited about the happy ending.
❤️ ➕ 🤗